scripts/pin_the_slow_way.py

```diff
@@ -32,6 +32,7 @@
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
           'geos': 'geos 3.4.*',
+          'giflib': 'giflib 5.1.2|5.1.2.*',
```

Comment:
According to that site they've never released a ``5.1.2.*`` type version? Can we guarantee if they did it wouldn't be breaking? Or should we just hard pin to `5.1.2`?